expr/function_test.go

```diff
@@ -0,0 +1,27 @@
+package expr_test
```

Comment:
Nit: Maybe call this functions_test.go to match functions.go.

Reply:
The compiler doesn't like that:
```
proc/cut.go:8:2: found packages expr (expr.go) and function (function_test.go) in /Users/aswan/src/zq/expr
```

Reply:
Sorry I wasn't clearer here. I meant that since the new function code is in a file named functions.go, you might want to change the name of this corresponding test file to functions_test.go for symmetry.